e2e/tests/localAuthorityMaintainsCaseAfterSDO_test.js

```diff
@@ -0,0 +1,61 @@
+const config = require('../config.js');
+const hearingDetails = require('../fixtures/hearingTypeDetails.js');
+const directions = require('../fixtures/directions.js');
+
+let caseId;
+
+Feature('Local authority manages case after SDO is issued');
+
+Before(async (I, caseViewPage, submitApplicationEventPage, sendCaseToGatekeeperEventPage, addHearingBookingDetailsEventPage, draftStandardDirectionsEventPage) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields();
+    await caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    await I.completeEvent('Submit');
+
+    // eslint-disable-next-line require-atomic-updates
```

Comment:
What is this for?

Reply:
I am not sure, haven't removed it as it was copy paste from other tests.